### DISCUSSION TITLE
不知道为什么很多默认样式都显示了NaN？？？？？？？

### DISCUSSION THREAD

**Discussion by pangxiaolong** [jdf2e/nutui-react#1257] [Q&A]
![image](https://github.com/jdf2e/nutui-react/assets/25050645/beb92518-0138-4421-9e1c-61caffc474d5)
比如这个tab的titie的高度和行高，很多组件自带样式都这样，导致我不得不重写样式class
我没有使用ConfigProvider

**Comment by oasis-cloud** [comment]
可以提交一个 issue 看看
